middleware/markdown/markdown.go

```diff
@@ -99,74 +70,67 @@ func (md Markdown) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error
 			continue
 		}
```

Comment:
Exiting the ```for``` early drops a level of indentation, and we don't really need the trap ```for``` in case a *return* is forgotten somewhere, or a *continue* slips through.

```go
var cfg *Config
for i := range md.Configs {
	if middleware…Matches { // not negated
		cfg = md.Configs[i]
		break // or goto
	}
}
if cfg == nil {
	return md.Next.ServeHTTP(w, r) // exit early
}
```